citus.spec

```
@@ -51,21 +61,22 @@ echo %{pginstdir}/lib/%{sname}.so >> installation_files.list
 [[ -f %{buildroot}%{pginstdir}/lib/citus_columnar.so ]] && echo %{pginstdir}/lib/citus_columnar.so >> installation_files.list
 echo %{pginstdir}/share/extension/%{sname}-*.sql >> installation_files.list
 echo %{pginstdir}/share/extension/%{sname}.control >> installation_files.list
+[[ -f %{buildroot}%{pginstdir}/bin/pg_send_cancellation ]] && echo %{pginstdir}/bin/pg_send_cancellation >> installation_files.list
 %ifarch ppc64 ppc64le
-  %else
-  %if 0%{?rhel} && 0%{?rhel} <= 6
-  %else
-    echo %{pginstdir}/lib/bitcode/%{sname}/*.bc >> installation_files.list
-    echo %{pginstdir}/lib/bitcode/%{sname}*.bc >> installation_files.list
-    echo %{pginstdir}/lib/bitcode/%{sname}/*/*.bc >> installation_files.list
-    
-    # Columnar does not exist in Citus versions < 10.0
-    # At this point, we don't have %{pginstdir},
-    # so first check build directory for columnar.
-    [[ -d %{buildroot}%{pginstdir}/lib/bitcode/columnar/ ]] && echo %{pginstdir}/lib/bitcode/columnar/*.bc >> installation_files.list
-    [[ -d %{buildroot}%{pginstdir}/lib/bitcode/citus_columnar/ ]] && echo %{pginstdir}/lib/bitcode/citus_columnar/*.bc >> installation_files.list
-    [[ -d %{buildroot}%{pginstdir}/lib/bitcode/citus_columnar/safeclib ]] && echo %{pginstdir}/lib/bitcode/citus_columnar/safeclib/*.bc >> installation_files.list
-  %endif
+%else
+    %if 0%{?rhel} && 0%{?rhel} <= 6
+    %else
+      echo %{pginstdir}/lib/bitcode/%{sname}/*.bc >> installation_files.list
+      echo %{pginstdir}/lib/bitcode/%{sname}*.bc >> installation_files.list
+      echo %{pginstdir}/lib/bitcode/%{sname}/*/*.bc >> installation_files.list
+
+      # Columnar does not exist in Citus versions < 10.0
+      # At this point, we don't have %{pginstdir},
+      # so first check build directory for columnar.
+      [[ -d %{buildroot}%{pginstdir}/lib/bitcode/columnar/ ]] && echo %{pginstdir}/lib/bitcode/columnar/*.bc >> installation_files.list
+      [[ -d %{buildroot}%{pginstdir}/lib/bitcode/citus_columnar/ ]] && echo %{pginstdir}/lib/bitcode/citus_columnar/*.bc >> installation_files.list
+      [[ -d %{buildroot}%{pginstdir}/lib/bitcode/citus_columnar/safeclib ]] && echo %{pginstdir}/lib/bitcode/citus_columnar/safeclib/*.bc >> installation_files.list
+    %endif
```

Can we use a consistent indentation? Maybe 4 spaces?

```suggestion
%else
    %if 0%{?rhel} && 0%{?rhel} <= 6
    %else
        echo %{pginstdir}/lib/bitcode/%{sname}/*.bc >> installation_files.list
        echo %{pginstdir}/lib/bitcode/%{sname}*.bc >> installation_files.list
        echo %{pginstdir}/lib/bitcode/%{sname}/*/*.bc >> installation_files.list

        # Columnar does not exist in Citus versions < 10.0
        # At this point, we don't have %{pginstdir},
        # so first check build directory for columnar.
        [[ -d %{buildroot}%{pginstdir}/lib/bitcode/columnar/ ]] && echo %{pginstdir}/lib/bitcode/columnar/*.bc >> installation_files.list
        [[ -d %{buildroot}%{pginstdir}/lib/bitcode/citus_columnar/ ]] && echo %{pginstdir}/lib/bitcode/citus_columnar/*.bc >> installation_files.list
        [[ -d %{buildroot}%{pginstdir}/lib/bitcode/citus_columnar/safeclib ]] && echo %{pginstdir}/lib/bitcode/citus_columnar/safeclib/*.bc >> installation_files.list
    %endif
```